src/cnxml/de.ts

```diff
@@ -85,6 +85,8 @@ export default function deserialize(
     }
 
     const doc: Doc = {
+        className: root.getAttribute('class') ?? undefined,
+        language: root.getAttributeNS('http://www.w3.org/XML/1998/namespace', 'lang') ?? undefined,
```

Comment:
Language is already being deserialized in lines 98-100 (100-102 after your changes).